packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -109,7 +116,7 @@ const ActionHeader = ({
 
   let closeButtonSmall;
   let backButtonSmall;
-  if (onClose && !onBack) {
+  if (onClose && !onBack && !keepCloseButton) {
```

Comment:
This approach works, but when this prop is set it removes the need for the responsive element. So a better solution would be to just render the actionHeader without the responsive element when this prop is set, so that we don't have that resize observer when we don't need it.

Reply:
Actually, nevermind. It's also needed for the expand/collapse logic.